apps/web/components/kanban/KanbanBoard.tsx

```
@@ -0,0 +1,368 @@
+      if (!hasDraggableData(active)) return;
+      if (active.data.current?.type === "Column") {
+        const startColumnIdx = columnsId.findIndex((id) => id === active.id);
+        const startColumn = columns[startColumnIdx];
+        return `Picked up Column ${startColumn?.title} at position: ${
+          startColumnIdx + 1
+        } of ${columnsId.length}`;
+      } else if (active.data.current?.type === "Task") {
+        pickedUpTaskColumn.current = active.data.current.task.columnId;
+        const { tasksInColumn, taskPosition, column } = getDraggingTaskData(
+          active.id,
+          pickedUpTaskColumn.current!, // TODO: !
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)